controller/controller.go

```diff
@@ -172,106 +172,112 @@ func (c *controller) updateIngresses() error {
 	var entries []IngressEntry
 	for _, ingress := range ingresses {
```

Comment:
Should we also wrap this processing in a try catch?

Reply:
try/catch doesn't exist in golang, not sure how to improve the function